src/twitter-adapter-playwright/TwitterBrowserAdapter.ts

```diff
@@ -1,7 +1,7 @@
 import { BrowserContext } from "playwright";
 import { TwitterBrowser } from "@/twitter-api/ports/TwitterBrowser";
-import { UserProfileViewAdapter } from "./UserProfileViewAdapter";
-import { UserTweetsViewAdapter } from "./UserTweetsViewAdapter";
+import { UserProfileViewAdapter } from "./UserProfileViewAdapter.js";
+import { UserTweetsViewAdapter } from "./UserTweetsViewAdapter.js";
```

Comment:
This is related to ES modules, and makes code a bit arbitrary

Reply:
And [breaks tests](https://github.com/despan/machina/actions/runs/3189696398/jobs/5203807621).